ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift

```diff
@@ -0,0 +1,381 @@
+              .foregroundStyle(Color(braveSystemName: .iconDefault))
+          }
+        }
+        .padding()
+        
+        if index != items.count {
+          Color(braveSystemName: .dividerSubtle)
+            .frame(height: 1.0)
+        }
+      }
+    }
+    .background {
+      RoundedRectangle(cornerRadius: 8.0, style: .continuous)
+        .stroke(Color(braveSystemName: .iconDefault), lineWidth: 0.5)
+    }
+    .background(Color(braveSystemName: .containerBackground))
+    .clipShape(RoundedRectangle(cornerRadius: 8.0, style: .continuous))
+    .shadow(color: .black.opacity(0.25), radius: 10.0, x: 0.0, y: 2.0)
+  }
+}
+
+private struct AIChatDropdownView: View {
+  enum Options: String, CaseIterable, Identifiable {
+    case notHelpful
+    case notWorking
+    case other
+    
+    var id: Self { self }
+
+    var rawValue: String {
+      switch self {
+      case .notHelpful:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      case .notWorking:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      case .other:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      }
+    }
+  }
+  
+  @Binding
+  var selectedIndex: Int
+  
+  @State
+  private var showMenu = false
+
+  var body: some View {
+    VStack {
+      HStack(spacing: 0.0) {
+        Text(Strings.AIChat.feedbackOptionsViewTitle)
+          .font(.caption)
+          .fontWeight(.semibold)
+          .foregroundStyle(Color(braveSystemName: .textPrimary))
+        
+        Text("*")
+          .font(.caption)
+          .fontWeight(.semibold)
+          .foregroundStyle(Color(braveSystemName: .systemfeedbackErrorText))
+      }
+      .frame(maxWidth: .infinity, alignment: .leading)
+      
+      DropdownView(showMenu: $showMenu) {
+        AIChatDropdownButton(action: {
+          showMenu.toggle()
+        }, title: Options.allCases[selectedIndex].rawValue)
+      } menuView: {
+        AIChatDropdownMenu(selectedIndex: $selectedIndex, items: Options.allCases)
+          .offset(x: 0.0, y: 12.0)
+          .onChange(of: selectedIndex) { _ in
+            showMenu = false
+          }
+      }
+    }
+  }
+}
+
+private struct AIChatFeedbackInputView: View {
+  @ObservedObject
+  var model: AIChatSpeechRecognitionModel
+  
+  @Binding
+  var text: String
+  
+  var body: some View {
+    HStack {
+      ZStack(alignment: .leading) {
+        TextEditor(text: $text)
+          .font(.subheadline)
+          .foregroundStyle(Color(braveSystemName: .textPrimary))
+          .autocorrectionDisabled()
+          .autocapitalization(.none)
+          .submitLabel(.done)
+          .frame(minHeight: 80.0)
+          .osAvailabilityModifiers({ view in
+            if #available(iOS 16.4, *) {
+              view
+                .scrollContentBackground(.hidden)
+            } else {
+              view.onAppear {
+                UITextView.appearance().backgroundColor = .clear
```

Comment:
This is going to apply to all UITextView's in the application, better to use the `Introspection` library to grab the actual `UITextView` and set the background color that way